draft-ietf-tls-subcerts.md

```diff
@@ -400,23 +401,24 @@ controls, physical security or hardware security modules.
 Delegated credentials do not provide any additional form of early revocation.
 Since it is short lived, the expiry of the delegated credential would revoke
 the credential.  Revocation of the long term private key that signs the
-delegated credential also implictly revokes the delegated credential.
+delegated credential also implicitly revokes the delegated credential.
 
 
 ## Privacy considerations
 
 Delegated credentials can be valid for 7 days and it is much easier for a
-service to create delegated credential than a certificate signed by a CA. A
+service to create delegated credential than a certificate signed by a CA.  A
 service could determine the client time and clock skew by creating several
-delegated credentials with different expiry timestamps and observing whether
-the client would accept it.  Client time could be unique and thus privacy
-sensitive clients, such as browsers in incognito mode, who do not trust the
-service might not want to advertise support for delegated credentials or limit
-the number of probes that a server can perform.
+delegated credentials with different expiry timestamps and observing whether the
+client would accept it.  Client time could be unique and thus privacy sensitive
+clients, such as browsers in incognito mode, who do not trust the service might
```

Comment:
who do not trust the service, ...

Reply:
Yeah, I'm not rightly sure what this means. Note that this text was here before.

Reply:
ah ok no problem, ill fix it in post